docs/source/tutorials/GetMetadataNCBI.md

```diff
@@ -0,0 +1,120 @@
+query to an NCBI database. A query can eg. be a specific taxon: species, subspecies.
+But a query can be more than that.
+
+The tool we will be using is : [NCBImeta](https://github.com/ktmeaton/NCBImeta)
+
+Example local installation
+```
+conda create -n ncbimeta -c bioconda ncbimeta
+```
+
+It is already installed on SAGA, to use it:
+```
+conda activate ncbimeta
+```
+
+To use ncbimeta, you will need to use an API (application programming interface) key from NCBI.
+An API key is an authentication code that will serves to tell NCBI that you are the user
+making requests to download metadata via ncbimeta software. Using an NCBI API key
+will give you the possibility to send many more queries to NCBI (eg. to download more metadata)
+per second than you would be able to perform without it.
+
+To get an API key from NCBI :
+- login (or create an account)
+- account > account settings : there you can create an API key you will use - copy it and keep it safe
+
+We will need to create a configuration file (.yalm) for each metadata download
+we will ask the software to do. The configuration file is not a configuration file to
+run the software, but it is a configuration file that specifies the search/query
+we are asking the software to do: to retrieve the available metadata we are interested in,
+and then the specification of the specific of fields we want to download in a database/table.  
+
+A trick is to create a simple search in NCBI ex:
+![screenshot1](./searchfield1.png)
+
+This search field will help you build your "metadata download configuration file" (.yalm).
+
+Now we do a specific search for a specific sample we are interested in to get the
+metadata for, eg. BioSample: `SAMN12536438`. This page give us the information
+for the metadata we can have access to and how it is structured in the database.
+To ease the process of creating the `.yalm` file we save this data as `xml` and
+open it with a web-browser so we can get the structure of the fields we want
+information from. ![screenshot2](./exportsearch_xml.png)
+
+This gives: [xml_file](./biosample_result.xml).
+
+Compare it to the [.yalm configuration file](./providencia_metadata.yaml)
+
+The `.yalm` file allow to define the destination of the download (OUTPUT_DIR),
+your NCBI identification (EMAIL and API_KEY), the name of the sqlite database it will
+download the metadata to (DATABASE), the tables it will create within the database
+(TABLES) and the columns/fields it will create and download data to for each
+table in the sqlite database (TABLE_COLUMNS, indentation: table name you want,
+indentation: the field/column names and the metadata that will be downloaded in each)
+
+So, you see that you can define your own fields, that were not provided in the other examples found in the [Biosample example](https://github.com/ktmeaton/NCBImeta/tree/master/schema).
+
+To define those fields you need to access at which hierarchy of the `.xml` file you downloaded as helper.
```

Comment:
This sentence does not make sense to me.

Reply:
And on reread I am completely lost here. I have no idea of how to get to the yaml file I need for the search.